main.tf

```diff
@@ -0,0 +1,41 @@
+# Using tf_ansible module
+
+resource "aws_key_pair" "default" {
```

Comment:
Do not define a new key pair. Require the user to pass in the key pair. Typically each server does not have it's own key pair.